### PR TITLE
Merge redesign onto develop

### PR DESCRIPTION
**Pull request by alfetopito**
Merging `feature/ui-redesign` branch onto `develop` plus fixing build.

Mostly linter errors due to unused vars and typescript things.

Not perfect but looking pretty nice!

Running locally the UI is functional, I'm able to place orders, create liquidity and see my balance.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr553--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/TokenRow.tsx [hunk 1]
```diff
@@ -192,15 +302,21 @@ const TokenRow: React.FC<Props> = ({
...
+        <span>
+          <button>+ Deposit</button>
+          <span>
+            Balance:
+            <WalletDetail>
+              {' '}
+              {balance ? formatAmount(balance.totalExchangeBalance, balance.decimals) : '0'}
+              {validateMaxAmount && <a onClick={useMax}>max</a>}
+            </WalletDetail>
+            <i aria-label="Tooltip"></i>
```
Comment (Velenir) [inline src/components/TradeWidget/TokenRow.tsx]:
```suggestion
            <i aria-label="Tooltip"/>
```

File: src/components/TradeWidget/index.tsx [hunk 2]
```diff
@@ -237,46 +412,78 @@ const TradeWidget: React.FC = () => {
...
+            tabIndex={5}
+          >
+            {isSubmitting && <FontAwesomeIcon icon={faSpinner} size="lg" spin={isSubmitting} />}{' '}
+            {sameToken ? 'Please select different tokens' : 'Submit limit order'}
           </SubmitButton>
         </WrappedForm>
       </FormContext>
+      <OrdersPanel>
+        {/* Toggle panel visibility (arrow) */}
+        <button></button>
```
Comment (Velenir) [inline src/components/TradeWidget/index.tsx]:
```suggestion
        <button/>
```

Emptybutton? Wouldn't it be better to add whatever is displayed in css (`:after or background-image` or whatever) into code?

File: src/components/TradeWidget/index.tsx [hunk 3]
```diff
@@ -237,46 +412,78 @@ const TradeWidget: React.FC = () => {
...
+        {/* Actual orders content */}
+        <div>
+          <h5>Your orders</h5>
+          <OrdersWidget></OrdersWidget>
```
Comment (Velenir) [inline src/components/TradeWidget/index.tsx]:
```suggestion
          <OrdersWidget/>
```

File: src/components/OrdersWidget/OrderRow.tsx [hunk 4]
```diff
@@ -106,26 +101,16 @@ const OrderDetails: React.FC<OrderDetailsProps> = ({ buyToken, sellToken, order,
...
-          <Highlight color={pending ? 'grey' : 'red'}>{price}</Highlight>{' '}
-          <strong>{displayTokenSymbolOrLink(buyToken)}</strong>
-        </div>
-      </div>
-      <div className="order-details-responsive">
-        <div className="order-details-subgrid">
-          <Highlight color={pending ? 'grey' : 'red'}>{price}</Highlight>{' '}
-          <strong>{displayTokenSymbolOrLink(buyToken)}</strong>
-        </div>
+        {/* <Highlight color={pending ? 'grey' : ''}></Highlight> */}
```
Comment (Velenir) [inline src/components/OrdersWidget/OrderRow.tsx]:
```suggestion
        {/* <Highlight color={pending ? 'grey' : ''}/> */}
```

Comment (alfetopito) [reply src/components/OrdersWidget/OrderRow.tsx]: C'mon, this is commented out!

File: src/components/TradeWidget/TokenRow.tsx (hunk 1 above)
Comment (alfetopito) [reply src/components/TradeWidget/TokenRow.tsx]: https://github.com/gnosis/dex-react/pull/554

File: src/components/TradeWidget/index.tsx (hunk 2 above)
Comment (alfetopito) [reply src/components/TradeWidget/index.tsx]: cc @biocom

File: src/components/OrdersWidget/OrderRow.tsx (hunk 4 above)
Comment (Velenir) [reply src/components/OrdersWidget/OrderRow.tsx]: :pensive:

**Pull request merged**